src/rhsm/config.py

```diff
@@ -199,20 +201,29 @@ def set(self, section, name, value):
                 self.add_section(section)
             super(RhsmConfigParser, self).set(section, name, value)
         if section == "logging" and name == "default_log_level":
-            self.is_log_level_valid(value)
+            if not self.is_log_level_valid(default_log_level) and self.is_log_level_valid(value):
+                print("The default_log_level is being set to a valid value '{value}'.  There should be no further action needed.".format(value=value),
+                      file=sys.stderr)
+            elif self.is_log_level_valid(default_log_level) and not self.is_log_level_valid(value):
+                self.print_invalid_log_level_msg(value)
+            elif not self.is_log_level_valid(default_log_level) and not self.is_log_level_valid(value):
+                print("The default_log_level is still being set to an invalid value '{value}', please try again.".format(value=value),
+                      file=sys.stderr)
 
     def is_log_level_valid(self, value):
-        valid = ['CRITICAL', 'ERROR', 'WARNING', 'INFO', 'DEBUG', 'NOSET']
+        valid = ['CRITICAL', 'ERROR', 'WARNING', 'INFO', 'DEBUG', 'NOTSET']
```

Comment:
please factorize this as class constant, to avoid duplicating it later on in `print_invalid_log_level_msg()`

Reply:
This is how I had it to begin with, I am changing it back